src/client/components/features/OffreDeStage/Déposer/Étape2Stage/StageDeposerOffreFormulaireÉtape2Stage.tsx

```diff
@@ -53,16 +62,172 @@ const dureeStageList: Option[] = [
 	{ libellé: '6 mois', valeur: (6 * DUREE_MOIS_EN_JOUR).toString() },
 ];
 
+function RadioDateDeDebut(props: { checked: boolean, onChange: () => void }) {
```

Comment:
```suggestion
function RadioConnaisOuNonDateDeDebutPrecise(props: { checked: boolean, onChange: () => void }) {
```

ou qqch dans ce genre, qui permet de savoir ce que c'est dans 2mois sans avoir à aller dans le composant